prajwal-vw-header/includes/css/scss/_module-header.scss

```
@@ -1,78 +1,11 @@
 .headerWrapper {
-  display: grid;
-  grid-template-columns: 1fr $container-width 1fr;
-  align-items: center;
-  background-color: rgba(0, 0, 0, 0.5);
-  position: absolute;
-  width: 100%;
-  transition: background-color 250ms ease-in-out;
-
-  &:hover {
-    background-color: $white;
-
-    &::before,
-    &::after {
-      border-top: 2px solid $black;
-    }
-
-    .brandLogoDealerNameWrapper {
-      &::before,
-      &::after {
-        border-top: 2px solid $black;
-      }
-    }
-
-    .brandLogo {
-      fill: $black;
-    }
-
-    .dealerName {
-      color: $black;
-    }
-  }
-}
-
-.headerWrapper::before,
-.headerWrapper::after {
-  content: "";
-  border-top: 2px solid white;
-}
-
-.brandLogoDealerNameWrapper {
-  display: grid;
-  width: 100%;
-  align-items: center;
-  text-align: center;
-  grid-template-columns: minmax(20px, 1fr) auto minmax(20px, 30fr);
-  grid-gap: 20px;
-  margin: 35px 0;
-  color: #fff;
-}
-
-.brandLogoDealerNameWrapper::before,
-.brandLogoDealerNameWrapper::after {
-  content: "";
-  border-top: 2px solid $white;
-}
-
-.brandLogoDealerNameLink {
-  text-decoration: none;
-}
-
-.logoName {
-  display: flex;
-  align-items: center;
-}
-
-.brandLogo {
-  width: 48px;
-  height: 48px;
-  fill: $white;
-}
-
-.dealerName {
-  @include primaryFontStackBold;
-  font-size: 20px;
-  color: $white;
-  padding-left: 10px;
+	background-color: $header-bg-color;
+	position: absolute;
+	width: 100%;
+	padding: 20px 0;
+	transition: background-color 250ms ease-in-out;
+
+	&:hover {
+		background-color: $white;
```

If you're going to control the `background-color` with a `variable` you should control the `hover` `background-color` with a `variable` as well so you can control both in the same place (`_variables.scss`)

```suggestion
		background-color: $header-bg-color-hover;
```

---

Resolved based on commit 8cd5ea5